Documentation/Deployment.md

```diff
@@ -0,0 +1,3 @@
+# Deployment
+
+We're using [Fastlane](https://fastlane.tools) to deploy the App to App Store Connect. Please see the [Fastlane README](../fastlane/README.md) on the available commands.
```

Comment:
Maybe we (okay, you) could add a little paragraph what needs to be done for new developers (register their devices in `fastlane/devices.txt`...)

Reply:
Yes, this file specifically is only meant for App Store deployments, but I'll add a bit more info on local deployment as well.